### PR TITLE
feat: extend the time we resume the session without showing OTP

### PR DESCRIPTION
**Pull request by omridan159**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/coding_guidelines/CODING_GUIDELINES.md)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add the appropiate QA label when dev review is completed
    - `needs-qa`: PR requires manual QA.
    - `No QA/E2E only`: PR does not require any manual QA effort. Prior to merging, ensure that you have successful end-to-end test runs in Bitrise. 
    - `Spot check on release build`: PR does not require feature QA but needs non-automated verification. In the description section, provide test scenarios. Add screenshots, and or recordings of what was tested.
5. Add `QA Passed` label when QA has signed off (Only required if the PR was labeled with `needs-qa`)
6. Add your team's label, i.e. label starting with `team-` (or `external-contributor` label if your not a MetaMask employee)

**Description**

Extended the session persistence time to one hour without showing OTP.

For the first hour after connecting, the session will remain active without requesting an OTP.

All of the examples SDK apps that use the OTP modal tested on IOS and Android with a new wallet build that includes the new OTP changes.

The cases that have been checked were `when the channel was active recently` and `when the channel was NOT active recently` in order to test both flows.

Everything appears to be working as expected.
**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

fixes #???

**Checklist**

* [x] There is a related GitHub issue
* [x] Tests are included if applicable
* [x] Any added code is fully documented

### CODE REVIEW

**Review by omridan159 — COMMENTED** [review]
I have read the CLA Document and I hereby sign the CLA

File: app/core/SDKConnect/SDKConnect.ts [hunk 1]
```diff
@@ -424,7 +448,7 @@ export class Connection extends EventEmitter2 {
         // Wait for bridge to be ready before handling messages.
         // It will wait until user accept/reject the connection request.
         try {
-          await this.checkPermissions(message);
+          await this.checkPermissions({ _message: message });
```
Comment (abretonc7s) [inline app/core/SDKConnect/SDKConnect.ts]: better rename to message instead _message

**Review by abretonc7s — DISMISSED** [review]
LGTM

**Review by abretonc7s — DISMISSED** [review]

**Review by omridan159 — COMMENTED** [review]
I have read the CLA Document and I hereby sign the CLA

**Review by omridan159 — COMMENTED** [review]
I have read the CLA Document and I hereby sign the CLA

**Review by omridan159 — COMMENTED** [review]
> I have read the CLA Document and I hereby sign the CLA

**Comment by omridan159** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by abretonc7s — DISMISSED** [review]
LGTM

**Review by abretonc7s — DISMISSED** [review]

**Review by andreahaku — DISMISSED** [review]
LGTM

**Review by andreahaku — APPROVED** [review]
LGTM

**Pull request merged**